src/components/structures/RightPanel.js

```diff
@@ -31,7 +31,6 @@ module.exports = React.createClass({
     displayName: 'RightPanel',
 
     propTypes: {
-        userId: React.PropTypes.string, // if showing an orphaned MemberInfo page, this is set
```

Comment:
Presumably the long term plan here to do the same for roomId, in which case we should add a comment or else someone will change it back to using props again.